odata-csdl/14 Vocabulary and Annotation.md

```diff
@@ -614,6 +624,16 @@ qualified name of function followed by parentheses containing the comma-separate
 all overloads of a [Function](#Function)| 
 qualified name of function| 
 <pre>`MySchema.MyFunction`</pre>
+bound [Function](#Function) overload via container| 
+Navigation Property via container or Property via container, followed by bound Function overload| 
+<pre>`MySchema.MyEntityContainer/MyEntitySet` 
+<br>` /MyComplexProperty` 
```

Comment:
What about paths without a navigation property, to differentiate functions on multiple entity sets sharing the same entity type?